packages/legend-extension-dsl-mastery/src/graph-manager/protocol/pure/DSL_Mastery_PureProtocolProcessorPlugin.ts

```diff
@@ -257,6 +266,41 @@ export class DSL_Mastery_PureProtocolProcessorPlugin
           V1_buildDataProvider(elementProtocol, context);
         },
       }),
+      new V1_ElementBuilder<V1_MasteryRuntime>({
+        elementClassName: MASTERY_RUNTIME_ELEMENT_CLASS_NAME,
+        _class: V1_MasteryRuntime,
+        firstPass: (
+          elementProtocol: V1_PackageableElement,
+          context: V1_GraphBuilderContext,
+        ): PackageableElement => {
+          const extraMasteryRuntimeBuilders =
+            context.extensions.plugins.flatMap(
+              (plugin) =>
+                (
+                  plugin as DSL_Mastery_PureProtocolProcessorPlugin_Extension
+                ).V1_getExtraMasteryRuntimeFirstPassBuilders?.() ?? [],
+            );
+          for (const builder of extraMasteryRuntimeBuilders) {
+            const metamodel = builder(
+              elementProtocol as V1_MasteryRuntime,
+              context,
+            );
+            if (metamodel) {
+              return metamodel;
+            }
+          }
+          throw new UnsupportedOperationError(
+            `Can't build runtime '${typeof elementProtocol}'`,
```

Comment:
Can you please modify this error message to `Can't build mastery runtime '${typeof elementProtocol}'`